packages/ripple-binary-codec/src/enums/README.md

```diff
@@ -53,8 +57,94 @@ See:
 - https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TER.h
 - https://xrpl.org/transaction-results.html
 
-TODO: Write a script to read rippled's source file and generate the necessary mapping.
+To generate a new definitions file from rippled source code, use this tool: https://github.com/RichardAH/xrpl-codec-gen
 
 ## Transaction Types
 
 See https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TxFormats.h
+
+# Defining Your Own Definitions
+
+If you're building your own sidechain or writing an amendment for the XRPL, you may need to create new XRPL definitions.
+
+To do that there are a couple things you need to do:
+
+1. Generate your own `definitions.json` file from rippled source code using [this tool](https://github.com/RichardAH/xrpl-codec-gen) (The default `definitions.json` for mainnet can be found [here](https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-binary-codec/src/enums/definitions.json))
+2. Create new SerializedType classes for any new Types (So that encode/decode behavior is defined).
+
+- Note: This is very rarely required. The SerializedType classes correspond to "ST..." classes in Rippled.
```

Comment:
```suggestion
2. Create new SerializedType classes for any new Types (So that encode/decode behavior is defined). The SerializedType classes correspond to "ST..." classes in Rippled. Note: This is very rarely required. 
```